src/appmixer/linear/core/FindIssues/FindIssues.js

```
@@ -0,0 +1,110 @@
+                                id
+                                name
+                                color
+                            }
+                        }
+                    }
+                }
+            }
+        `;
+
+        const variables = {};
+        if (query && query.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)